code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -335,12 +335,21 @@
 	turf_type = /turf/open/floor/plastic
 
 /obj/item/stack/tile/material
-	name = "tile"
+	name = "floor tile"
 	singular_name = "floor tile"
-	desc = "A tile of flooring."
+	throwforce = 5
 	icon_state = "material_tile"
 	turf_type = /turf/open/floor/material
-	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_UPDATE_DESC | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
+
+/obj/item/stack/tile/material/mat_update_desc(mat)
+	switch(throwforce)
```

Comment:
I think this could be better handled in Examine()

Reply:
also I feel like only adding this info to tiles might seem weird.

Reply:
I just wanted to make it consistent with the long-standing current iron floor tile description without misleading players when these tiles are made of a weak material.

Reply:
sure, just make sure this is in examine